docs/conf.py

```diff
@@ -8,6 +8,7 @@
 import subprocess
 from pathlib import Path
```

Comment:
I see the docs show "Copyright 2022" - you should be able to set it dynamically using `datetime` here;

```python
import datetime 
# ...
year = datetime.date.today().year
# ...
```

Reply:
good point!